content/docs/operating/security.md

```diff
@@ -116,9 +121,40 @@ resulting from additional load and failed scrapes.
 
 ## Authentication, Authorization, and Encryption
 
-Prometheus and its components do not provide any server-side
-authentication, authorization or encryption. If you require this, it is
-recommended to use a reverse proxy.
+In the future, server-side TLS support will be rolled out to the different
+Prometheus projects. Those projects include Prometheus, Alertmanager,
+Pushgateway and the official exporters.
+
+Authentication of clients by TLS client certs will also be supported.
+
+The Go projects will share the same TLS library, which will be based on the
+Go vanilla TLS library. We keep [Go default TLS
+parameters](https://golang.org/pkg/crypto/tls/#Config), with one exception: we
+support only TLSv1.2 and higher. This decision is based upon current industry
+standards (notes from [Google][tlsgoogle], [Apple][tlsapple],
+[Mozilla][tlsmozilla], and [Microsoft][tlsmicrosoft]).
+
+TLS will be added to Java exporters in the future.
+
+If you have special TLS needs, like a different cipher suite or older TLS
+version, the TLS support enables you to build a secure tunnel between the
+servers and reverse proxies with special settings.
+
+HTTP Basic Authentication will also be supported. Basic Authentication can be
+used without TLS, but it will then expose usernames and passwords in cleartext
+over the network.
+
+On the server side, basic authentication passwords are hashed on disk with the
+[bcrypt](https://en.wikipedia.org/wiki/Bcrypt) algorithm on disks, and it is
```

Comment:
"are hashed on disk with ... algorithm~~ on disks~~,"

Reply:
Thx for noticing.